docs/guides/configuration-guide/ceph.md

```diff
@@ -590,6 +590,46 @@ The Ceph dashboard is bootstrapped with the `ceph-bootstrap-dashboard` play.
 $ osism apply ceph-bootstrap-dashboard
 ```
 
+### Configuring the openstack loadbalancer to expose the ceph dashboard
+
+The ceph dashboard runs in an active/standby configuration. In its default standby instances will
+redirect to the active instance. Most deployments will want to use the openstack loadbalancer to
+expose the ceph dashboard on the internal network and direct traffic directly to the active
+instance.
+In this scenario the dashboard should be configured to return an http error with status `404` on
+standby instances.
+
+```yaml title="environments/ceph/configuration.yml"
+ceph_dashboard_standby_behaviour: error
+ceph_dashboard_standby_error_status_code: 404
+```
+
+Create a loadbalancer configuration
+
+```jinja2 title="environments/kolla/files/overlays/haproxy/services.d/ceph_dashboard.cfg"
+
+{% if openstack_version == "zed" or openstack_version == "2023.1" %}
+{%- set internal_tls_bind_info = 'ssl crt /etc/haproxy/haproxy-internal.pem' if kolla_enable_tls_internal|bool else '' %}
+{% else %}
+{%- set internal_tls_bind_info = 'ssl crt /etc/haproxy/certificates/haproxy-internal.pem' if kolla_enable_tls_internal|bool else '' %}
```

Comment:
Only use this line. We do not longer support OpenStack Zed or 2023.1

Reply:
Done